wtfis/models/abuseipdb.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from typing import Optional
+
+from pydantic import BaseModel, Field, RootModel
+
+
+class AbuseIpDb(BaseModel):
+    ip_address: str = Field(alias="ipAddress")
+    is_public: Optional[bool] = Field(None, alias="isPublic")
+    ip_version: Optional[int] = Field(None, alias="numDistinctUsers")
+    is_whitelisted: Optional[bool] = Field(None, alias="ipVersion")
+    abuse_confidence_score: int = Field(alias="abuseConfidenceScore")
+    country_code: Optional[str] = Field(None, alias="countryCode")
+    country_name: Optional[str] = Field(None, alias="countryName")
+    usage_type: Optional[str] = Field(None, alias="usageType")
+    isp: str
+    domain: Optional[str] = None
+    hostnames: Optional[list[str]] = None
```

Comment:
Should be `List[str]` and not `list[str]`. I think this is what's causing the "type object is not subscriptable" error in the tests.